### ISSUE TITLE
Can't install angularFire through bower.json

### ISSUE THREAD

**Issue by hswolff** [firebase/angularfire#51]
Hello folks!

This morning I went to include angularFire into my project via my `bower.json` file:

`"angular-fire": "~0.1.0"`

After doing so and running bower install from the CLI I got the following error:

```
/usr/local/share/npm/lib/node_modules/bower/node_modules/tmp/lib/tmp.js:260
  throw err;
        ^
TypeError: Invalid Version: 8bbe6022e28b28b8d697904415af89e39a49a0ef
    at new SemVer (/usr/local/share/npm/lib/node_modules/bower/node_modules/semver/semver.js:257:11)
    at compare (/usr/local/share/npm/lib/node_modules/bower/node_modules/semver/semver.js:410:10)
    at gte (/usr/local/share/npm/lib/node_modules/bower/node_modules/semver/semver.js:459:10)
    at cmp (/usr/local/share/npm/lib/node_modules/bower/node_modules/semver/semver.js:476:22)
    at Comparator.test (/usr/local/share/npm/lib/node_modules/bower/node_modules/semver/semver.js:546:10)
    at testSet (/usr/local/share/npm/lib/node_modules/bower/node_modules/semver/semver.js:818:17)
    at Range.test (/usr/local/share/npm/lib/node_modules/bower/node_modules/semver/semver.js:810:9)
    at Function.satisfies (/usr/local/share/npm/lib/node_modules/bower/node_modules/semver/semver.js:831:16)
    at null.<anonymous> (/usr/local/share/npm/lib/node_modules/bower/lib/core/package.js:773:23)
    at Array.filter (native)
```

I did some sleuthing around and it seems that there only exists one tag on this repo: `v0.1` - which is an incorrect semver format.

It seems if you were to re-tag it as `v0.1.0` it should work. 

Until then I'm working around this issue by just explicitly assigning what SHA to checkout in my bower.json file:

`"angular-fire": "d00e85750439e8fa2e338798ec6398836046f599"`

Thanks for your help!

**Comment by vinothbabu** [comment]
`bower install --save angularfire#0.82` silently fails. 

`bower not-cached    git://github.com/firebase/angularFire.git#0.82
bower resolve       git://github.com/firebase/angularFire.git#0.82
bower ENORESTARGET  No tag found that was able to satisfy 0.82

Additional error details:
Available versions: 0.9.1, 0.9.0, 0.8.2, 0.8.1, 0.8.0, 0.7.1, 0.7.0, 0.6.0, 0.6.
0-pre1, 0.5.0, 0.3.2, 0.3.1, 0.3.0, 0.2.0, 0.1.0`

**Comment by anantn** [comment]
You're saying "0.82" instead of "0.8.2". I recommend using the latest version anyway:

`bower install --save angularfire#0.9.1`
